mlrun/feature_store/retrieval/dask_merger.py

```diff
@@ -67,6 +67,7 @@ def _generate_vector(
 
             df = df.persist()
             dfs.append(df)
+            del df
```

Comment:
Does this help being placed here? You just added `df` to `dfs` and `dfs` is being passed on to `self.merge()`. So, will this really clear-up memory?

Reply:
As discussed, it's to avoid keeping the last df after merge and before write to targets.